package.json

```diff
@@ -41,39 +41,35 @@
   },
   "homepage": "https://github.com/Pocket/curated-corpus-api#readme",
   "dependencies": {
-    "@apollo/federation": "^0.33.9",
-    "@aws-sdk/client-eventbridge": "^3.56.0",
-    "@devoxa/prisma-relay-cursor-connection": "^2.1.0",
-    "@pocket-tools/apollo-utils": "^2.2.0",
-    "@prisma/client": "^3.9.1",
-    "@sentry/node": "^7.0.0",
-    "@sentry/tracing": "^7.0.0",
-    "@snowplow/node-tracker": "^3.3.0",
-    "@types/luxon": "^2.0.9",
-    "apollo-datasource-rest": "^3.5.0",
-    "apollo-server": "^3.6.3",
-    "apollo-server-cache-redis": "^3.3.1",
-    "apollo-server-core": "^3.6.3",
-    "apollo-server-express": "^3.6.3",
-    "apollo-server-plugin-base": "^3.5.1",
-    "apollo-server-plugin-response-cache": "^3.5.1",
-    "apollo-server-types": "^3.5.1",
-    "aws-sdk": "^2.1072.0",
-    "aws-xray-sdk-core": "^3.3.4",
-    "aws-xray-sdk-express": "^3.3.4",
-    "dataloader": "^2.0.0",
-    "date-fns": "^2.28.0",
-    "graphql": "^15.8.0",
-    "graphql-anywhere": "^4.2.7",
-    "graphql-scalars": "^1.14.1",
```

Comment:
cleaned up some of the needed packages and removed ones that were not imported/used